guides/contribution.md

```diff
@@ -0,0 +1,119 @@
+Whether you are a designer, developer, or product manager, your contributions can help us improve the system and make it more useful for a wider range of users.
+
+This guide is designed to help you understand how you can contribute to Mística, what kind of contributions we are looking for, and how to make sure your contributions are consistent with our way of working.
+
+We encourage you to read this guide carefully and get in touch with us if you have any questions or feedback.
+
+## Index
+
+- [Proposal](#proposals)
+- [Creation of an issue based on the proposal](#issues)
+- [Updating or creating the specs needed](#specifications)
+- [Development and final review](#from-design-to-development)
+
+## Contribution journey
+
+![Contribution journey](../.github/resources/guides/contribution/contribution_journey.svg)
+
+## Proposals
+
+Before submitting a proposal, we encourage you to review:
+
+* Our existing components and guidelines to ensure that your proposal aligns with our design principles and 
+* There's no other discussion already covering the same topic. You can find all of our design guidelines and documentation on [Telefónica's brand factory](https://brandfactory.telefonica.com/d/iSp7b1DkYygv/n-a#/get-started/what-is-mistica).
+* It's not covered by the existing [Component specs](https://www.figma.com/files/1125734703130062955/project/27955986/Component-Specs?fuid=1111936175780412673) 
+* The technical viability to include the proposal in your product (A validated PRD, or confirmation that backend can serve this information)
+
+To submit a proposal, create a [new discussion in the Mística Design GitHub repository](https://github.com/Telefonica/mistica-design/discussions).
+
+Your proposal should meet the following criteria:
+
+| Approval Criteria | Description                                                                                                                                                                                                                  |
+| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Global            | The proposal should be applicable to a broad range of use cases and scenarios, rather than being specific to a particular product or project. This ensures that the component is useful to a wider range of users and teams. |
+| Purpose           | The purpose of the proposed change needs to be well-explained. What the component is designed to solve, the benefits it provides, and any potential drawbacks or limitations.                                                |
+| Figma link        | You should provide a Figma link with examples of how the component could be used in different contexts. This helps us visualize the proposed change and understand its potential impact on the overall design system.        |
+
```

Comment:
:robot: OpenAI

The table provides clear approval criteria for proposals. No changes needed.
```
| Approval Criteria | Description                                                                                                                                                                                                                  |
| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Global            | The proposal should be applicable to a broad range of use cases and scenarios, rather than being specific to a particular product or project. This ensures that the component is useful to a wider range of users and teams. |
| Purpose           | The purpose of the proposed change needs to be well-explained. What the component is designed to solve, the benefits it provides, and any potential drawbacks or limitations.                                                |
| Figma link        | You should provide a Figma link with examples of how the component could be used in different contexts. This helps us visualize the proposed change and understand its potential impact on the overall design system.        |
```

<!-- This is an auto-generated comment by OpenAI -->